lib/bencode/~>0.8/bencode.rbi

```diff
@@ -0,0 +1,126 @@
+# typed: strong
+
+# bencode-0.8.2
+
+class Array
+  # Bencodes the Array object. Bencoded arrays are represented as +lxe+, where
```

Comment:
Thank you so much for comments!
They will show up in docs

Reply:
We just upgraded our sorbet to latest version today and found this out - it's amazing!!